### PR TITLE
Add session & participantFlags to participant

### PR DESCRIPTION
**Pull request by mario**
It allows showing if user is online + what it's status is (inCall / audio / video) without needing two separate calls.

### CODE REVIEW

**Comment by mario** [comment]
For some perspective, in order to show room info I currently need to fetch all participants and room which seems wasteful both time and resource wise on the phone.

File: lib/Controller/RoomController.php [hunk 1]
```diff
@@ -253,13 +253,23 @@ protected function formatRoom(Room $room, Participant $participant = null): arra
...
-				$participantList[(string) $user->getUID()] = [
+				$participantList[(string)$user->getUID()] = [
 					'name' => $user->getDisplayName(),
 					'type' => $data['participantType'],
 					'call' => $data['inCall'],
+					'sessionId' => $user->getSessionId(),
 				];
+
+				try {
+					$participant = $room->getParticipant($this->userId);
```
Comment (nickvergessen) [inline lib/Controller/RoomController.php]: this is certainly the wrong way

Comment (mario) [reply lib/Controller/RoomController.php]: Just noticed the bug and fixed it. Let me know if that's what you meant.

File: lib/Controller/RoomController.php [hunk 2]
```diff
@@ -253,13 +253,23 @@ protected function formatRoom(Room $room, Participant $participant = null): arra
...
-				$participantList[(string) $user->getUID()] = [
+				$participantList[(string)$user->getUID()] = [
 					'name' => $user->getDisplayName(),
 					'type' => $data['participantType'],
 					'call' => $data['inCall'],
+					'sessionId' => $user->getSessionId(),
 				];
+
+				try {
+					$participant = $room->getParticipant((string)$user->getUID());
```
Comment (nickvergessen) [inline lib/Controller/RoomController.php]: this does one query per user...
I will try to fix this and put it in the first response

**Comment by nickvergessen** [comment]
I split the getParticipants() method to 2 simpler versions. The remaining 4 calls I could not recover, should plan to get rid of them anytime in the future.

**Pull request merged**